lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -56,6 +56,7 @@ class CourseViewTestsMixin(object):
     def setUp(self):
         super(CourseViewTestsMixin, self).setUp()
         self.create_user_and_access_token()
+        self.create_course_data()
```

Comment:
Curious: Why was it necessary to move the `self.create_course_data()` call from `setUpClass` to `setUp`?  The former provided optimization in our unit tests (so the class didn't need to get recreated on each test, but just each test class).